zbarlight/__init__.py

```diff
@@ -1,5 +1,24 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
-from ._zbarlight import qr_code_scanner
+from ._zbarlight import zbar_code_scanner
+
+from PIL import Image
 
 __version__ = '0.1.1'
-__ALL__ = ['qr_code_scanner']
+__ALL__ = ['scan_codes', 'qr_code_scanner']
+
+
+def qr_code_scanner(image, width, height):
+    """Specific QR code scanner (deprecated)"""
+    result = zbar_code_scanner(b'qr.enable', image, width, height)
+    if isinstance(result, list) and len(result) != 1:
+        return None
+    return result[0] if result is not None else None
+
+
+def scan_codes(symbology, image):
+    """Generic code scanner"""
```

Comment:
Could we move the documentation of this function from the README to a docstring here? (The same goes for `qr_code_scanner()`.) That would avoid discrepancies and, well, having a docstring is better for those who read the source without the documentation on the side. :)

We would lose the ability to have the API in the README (unless we duplicate it), but an example may suffice with a link to ReadTheDocs for more.